activityDoc.md

```diff
@@ -24,32 +24,19 @@ Example:
 {
     "@context": [
       "https://www.w3.org/ns/activitystreams",
-      { "reader": "https://rebus.foundation/ns/reader" }
+      { reader: "https://rebus.foundation/ns/reader" }
     ],
-    "type": "Create",
-    "object": {
-      "type": "reader:Publication",
-      "name": "Publication A",
-      "attributedTo": [
-        {
-          "type": "Person",
-          "name": "Sample Author"
-        }
-      ],
-      "attachment": [
-        {
-          "type": "Document",
-          "name": "Chapter 9",
-          "position": 1,
-          "content": "Sample document content 1"
-        },
-        {
-          "type": "Document",
-          "name": "Chapter 2",
-          "position": 0,
-          "content": "Sample document content 2"
-        }
-      ]
+    type: "Create",
+    object: {
+      type: "Publication",
+      name: "Publication A",
+      author: ['John Smith'],
+      editor: 'Jane Doe',
+      description: 'Some description here',
+      links: [{ property: 'value' }],
```

Comment:
you should use an example that has real link objects in the links, readingOrder and resources. Check the integration tests for publications.